### PR TITLE
AWS High Availability Architecture

### LINKED ISSUE #23: AWS High Availability Architecture
**MicroByte Title:**  AWS High Availability Architecture

**Domain:**  Web Service

**Scope of learning for learners:**  
- Learner will be able to deploy a website with `High availability`, `secure`  website, and make website `manageable`.

**Activities:** 
- Activity 1 (Launched an instance in AWS): launched an instance in AWS using AWS CLI and this activity contain task for the learner.
- Activity 2 (Configured Webserver): configured Webserver by installing Apache Webserver and this activity contain task for the learner.
- Activity 3 (Created an EBS Volume): created EBS Volume using AWS CLI.
- Activity 4 (Attach EBS Volume to Instance): attached EBS Volume to the instance using AWS CLI and this activity contain task for the learner.
- Activity 5 (Created and Formated a Partition): created a Partition in Linux instance and then format the partition.
- Activity 6 (Mounted /var/www/html directory to partition): mounted  /var/www/html directory to the partition and created a HTML Page and this activity contain task for the learner.
- Activity 7 (Created S3 Bucket): created S3 bucket using AWS CLI.
- Activity 8 (Uploaded images in S3 Bucket):Uploaded images in S3 bucket and copy the URL of images and pasted in webpage.
- Activity 9(Created Cloudfront Distribution ): Created Cloudfront Distribution using AWS CLI and this activity contain task for the learner.

### PR DESCRIPTION
**Pull request by Nitesh-thapliyal**
### Checklist: 

- [x] Added activities for learners
- [x] Didn't give the solutions to the added activities straight away
- [x] Followed a LearnByDoing approach
- [x] Clean and well documented code (if source code files are involved)
- [x] Added images for better understanding if deemed necessary

### CODE REVIEW

**Comment by amoghrajesh** [comment]
Re - Earlier Issues:
Hey @Nitesh-thapliyal!
You have made most of the changes requested by me, but you might have missed out a few so I am gonna reiterate it for you:

In Prerequisites, you are assuming that it is a Windows user. What if he uses Linux/Mac? Provide links for these too.
You still have not mentioned what you need in the HTML in activity 7. Tell the learner how big the html should be (Just a simple boilerplate or does it need to have anything)
You got rid of the part in the previous version when you were launching the website in terms of activity 11 (Add that part but in Activity 10).
Apart from all these changes, very good work, man.
Makes these changes as soon as possible.

File: AWS High Availability Architecture/README.md
Comment (amoghrajesh) [inline AWS High Availability Architecture/README.md:167]: Just add here that: "Create an HTML file with your own content or if you can't think of content, use this one as below"

File: AWS High Availability Architecture/README.md
Comment (amoghrajesh) [inline AWS High Availability Architecture/README.md]: performing* - typo

File: AWS High Availability Architecture/README.md
Comment (amoghrajesh) [inline AWS High Availability Architecture/README.md]: What about Linux?

File: AWS High Availability Architecture/README.md
Comment (amoghrajesh) [inline AWS High Availability Architecture/README.md]: What does this image mean? Add a caption for it

**Review by amoghrajesh — CHANGES_REQUESTED** [review]
Hey @Nitesh-thapliyal!
This is pretty cool content and I like it personally.
I have requested very minor changes and after that, it is good to go from my end.
Thanks for your generous contribution & let me know once you make these.

**Review by amoghrajesh — APPROVED** [review]
@Nitesh-thapliyal perfect!
Thanks for your quick response to editing your work.
I like this content. Let's see what other reviewers have to say about this one.

P.S: You missed out on my last comment in the previous review. Correct it till other reviewers see this one.
Also, start to think of some learner issues if possible that can be raised here.

Thanks!

File: AWS High Availability Architecture/README.md
Comment (Kevinpaulose05) [inline AWS High Availability Architecture/README.md]: Typo- want's should be want(or something) in line 13

Comment (YourAKShaw) [reply AWS High Availability Architecture/README.md]: If you want to deploy a website and also want it to be fast, secure and available all the time, even in the time of some failure in the network, then `High Availability Architecture` comes in play.

Just copy paste this instead.

File: AWS High Availability Architecture/README.md
Comment (YourAKShaw) [inline AWS High Availability Architecture/README.md]: A learner who might be using Linux/Mac-OS would have a terminal. It seems that is instruction will be similar for terminal too, kindly write it that way. Writing only `window cmd` might be a little misleading, plus it's `Windows`.

**Review by YourAKShaw — CHANGES_REQUESTED** [review]
Hi @Nitesh-thapliyal! Overall, the content seems well laid out. I'm impressed. Kindly make the other small changes quick. And we'll soon give it a parting approval, merging it.

**Review by Kevinpaulose05 — APPROVED** [review]
Awesome job @Nitesh-thapliyal . Just one suggestion - It would be better if u could centre align the first image (of the world map). Two things to do **after merging**(u already know this :) ) - 
1. Open issues for learners so that they can contribute to Ur microbyte
2. Showcase Ur microbyte in slack's microbyte showcase channel using [this](https://github.com/Crio-Bytes/Demo-Repo/blob/master/Maintainer%20Resources/new-microbyte-showcase-template.md)

Happy Contributing!!

File: AWS High Availability Architecture/README.md
Comment (Kevinpaulose05) [reply AWS High Availability Architecture/README.md]: U can mention that the environment u have used is `windows` and so some things maybe be different for different OS

**Review by YourAKShaw — APPROVED** [review]

**Comment by YourAKShaw** [comment]
@all-contributors kindly add @Nitesh-thapliyal for:

- [x] code
- [x] content
- [x] ideas
- [x] doc

**Pull request merged**
